src/datasources/config-api/config-api.service.spec.ts

```diff
@@ -1,44 +1,7 @@
 import { ConfigApi } from './config-api.service';
 import { FakeConfigurationService } from '../../common/config/__tests__/fake.configuration.service';
 import { CacheFirstDataSource } from '../cache/cache.first.data.source';
-import { Chain } from '../../domain/chains/entities/chain.entity';
-import { Page } from '../../common/entities/page.entity';
-
-const CHAINS: Page<Chain> = {
-  count: 2,
-  results: [
-    {
-      chainId: '1',
-      chainName: 'Ethereum',
-      transactionService: 'https://safe-transaction.mainnet.gnosis.io',
-      vpcTransactionService:
-        'http://mainnet-safe-transaction-web.safe.svc.cluster.local',
-      nativeCurrency: {
-        name: 'Ether',
-        symbol: 'ETH',
-        decimals: 18,
-        logoUri:
-          'https://safe-transaction-assets.gnosis-safe.io/chains/1/currency_logo.png',
-      },
-    },
-    {
-      chainId: '100',
-      chainName: 'Gnosis Chain',
-      transactionService: 'https://safe-transaction.xdai.gnosis.io',
-      vpcTransactionService:
-        'http://xdai-safe-transaction-web.safe.svc.cluster.local',
-      nativeCurrency: {
-        name: 'xDai',
-        symbol: 'XDAI',
-        decimals: 18,
-        logoUri:
-          'https://safe-transaction-assets.gnosis-safe.io/chains/100/currency_logo.png',
-      },
-    },
-  ],
-};
-
-const CHAIN: Chain = CHAINS.results[0];
```

Comment:
I moved these to the tests because I find that using the factories is more readable (and also we avoid sharing state between the tests)

Reply:
Yes I agree, but could be better in a custom PR with this purpose, because if we need to revert this PR shouldn't revert this. It's only an idea.

Reply:
I've changed it here because this was not compiling either way due to the new schema.